### ISSUE TITLE
Minestrike minigame completely broken after MP update

### ISSUE THREAD

**Issue by bambam221** [ImpactDevelopment/ImpactIssues#1990]
Following the Mineplex Valentine's update the minestrike mode in the minigames got completely broken. Nothing is properly functioning the gamemode similar to when the mode broken after the Halloween update in October.

**Comment by LeafHacker** [comment]
@ZeroMemes maybe we need some api for stuff that changes a lot e.g. murder mystery items?

**Comment by ScanmanTM** [comment]
I was trying the minestrike module on 1.12.2 - 4.8.3 and the module isn't targeting players nor shooting at them.

**Comment by S-B99** [comment]
What do you think this issue is for? And the one I marked as duplicate?

**Comment by CrypteIsDefunct** [comment]
Any timeframe on a fix for this?

**Comment by dominikaaaa** [comment]
You must be new around here

**Comment by CrypteIsDefunct** [comment]
> You must be new around here

Yes in fact I am! :) I'm not quite sure how everything works yet. Should I not have asked that question?

**Comment by dominikaaaa** [comment]
No, there just isn't a milestone so there isn't an ETA

**Comment by MightyKnight** [comment]
@CrypteIsDefunct Questions are always appreciated and important for the community. Github just isn't the best Q&A platform.
For further questions I may direct you to the [Impact Discord](https://discord.gg/YFhR2Ab), you'll be welcomed by plenty of good supporters.

**Comment by CrypteIsDefunct** [comment]
MIghtyKnight Ok makes sense. Thank you!
